mwm-5-dynamic-delivery/presentation/PITCHME.md

```diff
@@ -298,28 +298,66 @@ interface SplitInstallManager {
 
 ---
 
-### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Download</span><span style="text-transform: none; font-size:0.8em;"> a dynamic module</span>
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Install</span><span style="text-transform: none; font-size:0.8em;"> a dynamic module</span>
 
 <br/>
 
 ```kotlin
-val manager = SplitInstallManagerFactory.create(context)
+val splitInstallManager = SplitInstallManagerFactory.create(context)
```

Comment:
Why this renaming?

Because on next slide I will not re-write this line with the factory. The name `splitInstallManager` seems maybe more explicit.